src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/LifecycleProperty.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Reflection;
+using Microsoft.Extensions.Internal;
+
+namespace Microsoft.AspNetCore.Mvc.ViewFeatures.Internal
+{
+    [DebuggerDisplay("{PropertyInfo, nq}")]
+    public readonly struct LifecycleProperty
```

Comment:
Can this move to MVC.Core in the infrastructure namespace?

Reply:
I don't see a lot of value in exposing a public API for `PropertyHelper`.

Reply:
On second thought, I'm ok with what you have here, because all of these really are "view features". We don't really care about moving them lower in the stack, and since this is .Internal, we could do taht any time we wanted to. 👍